egs/wsj/s5/steps/libs/nnet3/train/frame_level_objf/common.py

```diff
@@ -30,7 +29,7 @@ def train_new_models(dir, iter, srand, num_jobs,
                      shuffle_buffer_size, minibatch_size,
                      cache_read_opt, run_opts,
                      frames_per_eg=-1,
-                     min_deriv_time=None, max_deriv_time=None):
```

Comment:
@pegahgh , this is still not fixed.  Can you please go through all the comments again and make sure that you have addressed them?